fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanDecisionWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -508,6 +532,14 @@ public CommandProcessingResult acceptIcReviewDecisionLevelFour(Long loanId, Json
                     String.format("Loan Approval Matrix with Currency [ %s ] doesn't exist. Approval matrix is expected to continue ",
                             loan.getCurrencyCode()));
         }
+
+        /* TODO Get max loan amount from cash flow calculation */
+        final BigDecimal maxLoanAmount = BigDecimal.valueOf(100000);
+        final int comparisonResult = maxLoanAmount.compareTo(recommendedAmount);
+        if(comparisonResult < 0) {
+            throw new GeneralPlatformDomainRuleException("error.msg.loan.ic.review.recommended.amount.must.be.lower.than.max.loan.amount",
+                    String.format("IC Review recommended value must be lower than Loan maximum amount from cash flow calculation"));
```

Comment:
You used format without a placeholder

Reply:
Hi @oluexpert99 , this is temporary, right now it's static value once the value gets populated it will be coming from object